docs/api.rst

```diff
@@ -1178,3 +1181,1395 @@ The brokers resource provides access to the current state of Kafka brokers in th
+      Host: proxy-instance.kafkaproxy.example.com
+      Content-Type: application/vnd.kafka.v2+json      
+
+      {
+        "topics": [
+          "test1",
+	  "test2"
+        ]
+      }
+      
+   **Example response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 204 No Content
+
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      POST /consumers/testgroup/instances/my_consumer/subscription HTTP/1.1
+      Host: proxy-instance.kafkaproxy.example.com
+      Content-Type: application/vnd.kafka.v2+json      
+
+      {
+        "topic_pattern": "test.*"
+      }
+      
+   **Example response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 204 No Content
+
+
+.. http:get:: /consumers/(string:group_name)/instances/(string:instance)/subscription
+	       
+   Get the current subscribed list of topics
+
+   :param string group_name: The name of the consumer group
+   :param string instance: The ID of the consumer instance
+   :>jsonarr topics: A list of subscribed topics
+   :>jsonarr string topics[i]: Name of the topic
+
+   :statuscode 404:
+      * Error code 40403 -- Consumer instance not found
+   :statuscode 500:
+      * Error code 500 -- General consumer error response, caused by an exception during the
+        operation. An error message is included in the standard format which explains the cause.
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      GET /consumers/testgroup/instances/my_consumer/subscription HTTP/1.1
+      Host: proxy-instance.kafkaproxy.example.com
+      Accept: application/vnd.kafka.v2+json      
+
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/vnd.kafka.v2+json
+     
+      {
+        "topics": [
+          "test1",
+	  "test2"
+        ]
+      }
+
+.. http:delete:: /consumers/(string:group_name)/instances/(string:instance)/subscription
+
+   Unsubscribe from topics currently subscribed.
+
+   Note that this request *must* be made to the specific REST proxy instance holding the consumer
+   instance.
+
+   :param string group_name: The name of the consumer group
+   :param string instance: The ID of the consumer instance
+
+   :statuscode 404:
+      * Error code 40403 -- Consumer instance not found
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      DELETE /consumers/testgroup/instances/my_consumer/subscription HTTP/1.1
+      Host: proxy-instance.kafkaproxy.example.com
+      Accept: application/vnd.kafka.v2+json, application/vnd.kafka+json, application/json
+
+   **Example response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 204 No Content
+   
+
+.. http:post:: /consumers/(string:group_name)/instances/(string:instance)/assignments
+	       
+   Manually assign a list of partitions to this consumer.
+
+   :param string group_name: The name of the consumer group
+   :param string instance: The ID of the consumer instance
+
+   :<jsonarr partitions: A list of partitions to assign to this consumer
```

Comment:
Oh, and in fact it seems that the example below shows that there is a 204 with No Content. Does that reflect what the code does?